DashAI/back/api/api_v1/endpoints/global_explanation.py

```diff
@@ -0,0 +1,198 @@
+import logging
+from typing import Union
+
+from fastapi import APIRouter, Depends, status
+from fastapi.exceptions import HTTPException
+from sqlalchemy import exc, select
+from sqlalchemy.orm import Session
+
+from DashAI.back.api.api_v1.schemas.explanation_params import GlobalExplanationParams
+from DashAI.back.api.deps import get_db
```

Comment:
Change deps from dependency injection: 

```suggestion
from dependency_injection import inject, Provide
from DashAI.back.containers import Container
from DashAI.back.dependencies.database.models import Experiment, Run, RunStatus

```